t/e2e/whisper/whisper_mailbox_test.go

```diff
@@ -469,6 +457,29 @@ func (s *WhisperMailboxSuite) getMessagesByMessageFilterID(rpcCli *rpc.Client, m
 	return messages.Result
 }
 
+func (s *WhisperMailboxSuite) getMessagesByMessageFilterIDWithTracer(rpcCli *rpc.Client, messageFilterID string, tracer *envelopeTracer) (messages []map[string]interface{}) {
```

Comment:
I guess we wait for a single message. Wouldn't it be easier to track if a message with a particular hash got received? `tracer.envelopChan` should provide this information I guess.

Reply:
Easier, I don't think so. It's definitely easy to do though, but I'm not sure if we gain much, since we're just waiting for a signal that we're good to go before calling `getFilterMessages`. But I agree that it's more explicit, I'll look into that.

Reply:
@adambabik are you happy with the changes I made, or is there something missing?